app/(protected)/dashboard/(admin)/user/accounts/components/EditPermissionCell.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+
+import { ButtonBase, Box, Typography, styled, Menu, MenuItem, Divider } from "@mui/material";
+import { GridCellParams } from "@mui/x-data-grid";
+
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+import { Colors } from "@commons/enum";
+import { formatText } from "@commons/utility";
+import EditPermissionDialog from "./EditPermissionDialog";
+
+const CellTypography = styled(Typography)(({ theme }) => ({
+	fontWeight: 400,
+	fontSize: theme.spacing(1.75),
+	color: Colors.DARK_GREY,
+	marginRight: theme.spacing(0.5)
+}));
+
+interface IEditPermissionCell {
+	params: GridCellParams;
+}
+const EditPermissionCell: React.FC<IEditPermissionCell> = ({ params }) => {
+	const [openEditPermissionsDialog, setOpenEditPermissionsDialog] = useState(false);
+	const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+	const open = Boolean(anchorEl);
+
+	const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+		setAnchorEl(event.currentTarget);
+	};
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	const handleEditPermissionsDialog = () => {
+		setOpenEditPermissionsDialog(true);
+	};
+
+	const handleCloseEditPermissionsDialog = () => {
+		setOpenEditPermissionsDialog(false);
+	}
```

Comment:
can't we combine this? 
```suggestion
	const handleEditPermissionsDialog = (open: boolean) => {
	        return () => {
		         setOpenEditPermissionsDialog(open);
		}
	};
```

and update L#73-74

Reply:
done